src/scripts/main.js

```diff
@@ -22,7 +22,17 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  for (const key in data) {
+    const list = document.createElement('ul');
```

Comment:
you create a list on each iteration, but it's abundant (you need to create a li element on each cycle and then append it to the initial list)